### ISSUE TITLE
Revamp initial setup dialog to be more useful

### ISSUE THREAD

**Issue by joshuaboniface** [jellyfin/jellyfin#380]
The current initial setup screen is a great idea, however the execution falls flat. There's a lot of things that can be improved on it, in almost every screen. This issue will track the required changes.

Sub-issues: #192

**Comment by sparky8251** [comment]
I agree with these changes. Maybe just make it so whatever page is last has the "finish and go to dashboard" rather than a unique page, but thats something requiring testing.

**Comment by ploughpuff** [comment]
It would be nice if the "Preferred Metadata Language" (page 4) could somehow inherit the "Preferred display language" (page 1) as its default...
